telegram/ext/_application.py

```diff
@@ -564,9 +574,11 @@ async def stop(self) -> None:
             :meth:`start`
 
         Note:
-            This does *not* stop :attr:`updater`. You need to either manually call
-            :meth:`telegram.ext.Updater.stop` or use one of :meth:`run_polling` or
+            * This does *not* stop :attr:`updater`. You need to either manually call\
+            :meth:`telegram.ext.Updater.stop` or use one of :meth:`run_polling` or\
             :meth:`run_webhook`.
+            * Does *not* call :attr:`post_stop` - that is only done by\
+            :meth:`run_polling` and :meth:`run_webhook`.
```

Comment:
```suggestion
            * This does *not* stop :attr:`updater`. You need to either manually call
              :meth:`telegram.ext.Updater.stop` or use one of :meth:`run_polling` or
              :meth:`run_webhook`.
            * Does *not* call :attr:`post_stop` - that is only done by
              :meth:`run_polling` and :meth:`run_webhook`.
```

I did bullet lists in constants.py and discovered that as long as you add a 2 whitespace indent (to be in line with an asterisk), Sphinx will understand that and you won't need slashes.

Reply:
Thanks! :smiley: